content/50.discussion.md

```diff
@@ -11,8 +11,42 @@ In this review, we seek to explain the scientific rationale underlying these tec
 
 ### Concerns about Equity in Healthcare
 
-Scientific and medical research broadly is shaped by a number of biases.
-Some concerns include how clinical trials recruit and operate.
+The objective of the paper is to compile and synthesis literature on the disease COVID-19.
+To accurately and completely discuss this pandemic, health disparities and bias need to be addressed, so that the information is applicable to all.
+Bias in medicine can be characterized as when a physician and/or patient “others” an individual in a healthcare setting, oftentimes this occurs when a person looks or acts differently.
+By being self-aware of our own implicit biases, one can work towards equality.
```

Comment:
This is definitely a good point (because it applies to science as well as clinical trials and medical treatments), but I would suggest maybe reframing a little to bring it back to the context of this review (e.g., COVID-19 outcomes and medical research).
```

Reply:
I agree about the reframing -- these are good points, but they sound like a letter to the editor rather than a research paper.  Discussing racial bias in a review about COVID-19 is necessary because of the differential death rates observed, but putting a complete discussion of bias in the medical profession in a review about COVID-19 isn't feasible -- it just sounds like a digression.